optimize/self-managed/optimize-deployment/configuration/multi-tenancy.md

```diff
@@ -4,59 +4,60 @@ title: "Multi-tenancy"
+The `CAMUNDA_OPTIMIZE_IDENTITY_BASE_URL` environment variable has to be set to enable Optimize to retrieve tenant authorizations from Identity. If this base URL is not configured, Optimize will not be able to retrieve tenant authorizations and users will not be able to access any tenant's data in Optimize.
 
-If required the tenant authorization cache in Optimize can also be configured via these optional settings:
+If required, the tenant authorization cache in Optimize can also be configured via these optional settings:
 
-| YAML Path                                         | Environment Variable                                                           | Default Value | Description                                                        |
+| YAML path                                         | Environment variable                                                           | Default value | Description                                                        |
 | ------------------------------------------------- | ------------------------------------------------------------------------------ | ------------- | ------------------------------------------------------------------ |
-| caches.cloudTenantAuthorizations.maxSize          | CAMUNDA_OPTIMIZE_CACHES_CLOUD_TENANT_AUTHORIZATIONS_MAX_SIZE                   | 10000         | The maximum size of the C8 tenant authorizations cache.            |
+| caches.cloudTenantAuthorizations.maxSize          | CAMUNDA_OPTIMIZE_CACHES_CLOUD_TENANT_AUTHORIZATIONS_MAX_SIZE                   | 10000         | The maximum size of the Camunda 8 tenant authorizations cache.     |
 | caches.cloudTenantAuthorizations.defaultTtlMillis | CAMUNDA_OPTIMIZE_CACHES_CLOUD_TENANT_AUTHORIZATIONS_MIN_FETCH_INTERVAL_SECONDS | 300000        | The time in milliseconds the tenant authorizations will be cached. |
 
 ## Possible Camunda 7 multi-tenancy scenarios
 
 As described in the [Camunda 7 documentation](https://docs.camunda.org/manual/latest/user-guide/process-engine/multi-tenancy/), there are two possible multi-tenant scenarios which are also supported by Optimize:
 
-- [Possible multi-tenancy scenarios](#possible-camunda-7-multi-tenancy-scenarios)
-  - [Single process engine with tenant-identifiers](#single-camunda-7-process-engine-with-tenant-identifiers)
-  - [One process engine per tenant](#one-camunda-7-process-engine-per-tenant)
+- [Default tenant authorizations in Optimize](#default-tenant-authorizations-in-optimize)
+- [Configuration](#configuration)
+- [Possible Camunda 7 multi-tenancy scenarios](#possible-camunda-7-multi-tenancy-scenarios)
+- [Single Camunda 7 process engine with tenant-identifiers](#single-camunda-7-process-engine-with-tenant-identifiers)
+- [One Camunda 7 process engine per tenant](#one-camunda-7-process-engine-per-tenant)
```

Comment:
I think adding all of these sections here then clashes with the above sentence ".., there are two possible multi-tenant scenarios which are also supported by Optimize:"
We could adjust this, but I'm also thinking this page isn't that big that we need a separate list of contents. Shall we just remove it? @christinaausley

Reply:
I thought about this when I saw it -- we can remove the ToC here.

Reply:
@christinaausley adjusted, are we good to merge with that change?